packages/shared/components/modals/AccountSwitcherModal.svelte

```diff
@@ -2,27 +2,41 @@
     import { localize } from '@core/i18n'
     import { BASE_TOKEN } from '@core/network'
     import { activeProfile, visibleActiveAccounts } from '@core/profile'
-    import { sumBalanceForAccounts } from '@core/account'
+    import { selectedAccount, sumBalanceForAccounts } from '@core/account'
     import { formatTokenAmountBestMatch } from '@core/wallet'
     import { openPopup } from '@lib/popup'
     import { AccountSwitcherMenuItem } from 'shared/components/molecules'
     import { HR, Icon, Modal, Text } from 'shared/components'
+    import { tick } from 'svelte'
 
     export let modal: Modal
 
     $: totalBalance = sumBalanceForAccounts($visibleActiveAccounts)
 
+    function scrollToSelectedAccount() {
+        tick().then(() => {
+            const el = document.getElementById(`account-${$selectedAccount.id}`)
+            el?.scrollIntoView({ behavior: 'auto' })
+        })
```

Comment:
Is it possible to convert this to `async/await`? It would be great to not use callbacks at all :D